avocado/core/multiplexer.py

```diff
@@ -455,6 +465,8 @@ def data_merge(self, tree):
         :param tree: Tree to be merged into this database.
         :type tree: :class:`avocado.core.tree.TreeNode`
         """
+        if self._skip_new_data_check("data_merge", (tree,)):
```

Comment:
OK, although this looks like someting that shouldn't change behavior (besides the fact that it used the new `data_inject` "signature" with `name`), and could even be fast forwarded to master if you choose to.